integration_tests/tests/forseti/controls/inventory.rb

```diff
@@ -0,0 +1,95 @@
+require 'json'
+control 'inventory' do
+
+    describe "Inventory" do
+        before :context do
+            command("forseti inventory purge 0").result
+            command("forseti inventory create").result
+            command("sudo apt-get -y install mysql-client").result
```

Comment:
I have asked this before, please move this install of mysql-client into the vm setup, and outside of the test.

Reply:
I have moved `mysql-client` installation at the top so it is executed only once for all the tests. I will look into moving this outside of the test file.

Reply:
Done

Reply:
You can check out the build logs here -  https://travis-ci.org/forseti-security/forseti-security/jobs/590150397